### ISSUE TITLE
bug 

### ISSUE THREAD

**Issue by Johnnysausage** [nicotine-plus/nicotine-plus#2233]
Nicotine+ Version: 3.2.4
GTK Version: 3.24.34
Python Version: 3.10.5 (main, Aug  5 2022, 05:55:00)  [GCC 12.1.0 64 bit (AMD64)]

Type: <class 'ValueError'>
Value: string too long (274, maximum length 256)
Traceback:   File "D:/a/nicotine-plus/nicotine-plus/pynicotine/gtkgui/frame.py", line 1827, in _on_critical_error_threading
  File "D:/a/_temp/msys64/mingw64/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
  File "D:/a/_temp/msys64/mingw64/lib/python3.10/threading.py", line 953, in run
  File "D:/a/nicotine-plus/nicotine-plus/pynicotine/gtkgui/widgets/notifications.py", line 189, in work
  File "D:/a/nicotine-plus/nicotine-plus/pynicotine/gtkgui/widgets/notifications.py", line 202, in _notify

basically i turned my laptop back on and had a look at the downloads section to view what was being downloaded and it randomly brought up a bug box saying it will crash. anyway i ignored the message and copied the text, and nicotine is still running now

**Comment by slook** [comment]
This crash should have been fixed along time ago by https://github.com/nicotine-plus/nicotine-plus/pull/1836 but I don't understand why the ValueError is encountered when the code is there to avoid it...

_https://github.com/nicotine-plus/nicotine-plus/blob/3.2.4/pynicotine/gtkgui/widgets/notifications.py#L202_
https://github.com/nicotine-plus/nicotine-plus/blob/a414df623c465d575fba7ef71813f8f472381f98/pynicotine/gtkgui/widgets/notifications.py#L202

**Comment by slook** [comment]
Please rename this issue to something like "Notifications string too long ValueError crash on Windows"
